packages/library-base/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -67,6 +67,26 @@ interface MutableRealm : TypedRealm {
      */
     override fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T>
 
+    /**
+     * Returns a [RealmQuery] matching the predicate represented by [query].
+     *
+     * The results yielded by the query are live and thus also reflect any update to the
+     * [MutableRealm].
+     *
+     * Said results are only valid on the calling thread.
+     *
+     * The resulting query is lazily evaluated and will not perform any calculations until
+     * [RealmQuery.find] is called.
```

Comment:
Should we mention that calling `asFlow()` is illegal here? Or should we mention it on each `asFlow()` method in the   subclasses?